internal/services/desktopvirtualization/virtual_desktop_host_pool_resource.go

```diff
@@ -114,6 +115,12 @@ func resourceVirtualDesktopHostPool() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"public_network_access_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
```

Comment:
There appear to be more values supported here than just enabled/disabled.

Although convention in the provider so far has been to expose this as a boolean, due to the rollout of additional network security features we need to begin exposing this as a string property.

Can we please change this to 
```suggestion
			"public_network_access": {
				Type:         pluginsdk.TypeString,
				Optional:     true,
				Default:      hostpool.HostpoolPublicNetworkAccessEnabled,
				ValidateFunc: validation.StringInSlice(hostpool.PossibleValuesForHostpoolPublicNetworkAccess(), false),
			},
```